pages/common/alias.md

```diff
@@ -1,12 +1,16 @@
 # alias
 
-> Creates an alias for a word when used as the first word of a command.
-> Note that these operations are not permanent, expiring when the current shell session ends.
+> Registers words that are replaced by a command string (which can comprise multiple commands, shell functions, etc).
```

Comment:
I would prefer to keep the word alias somewhere. Maybe something like "Creates an alias for a word that are replaced by a command string (which can comprise multiple commands, shell functions, etc)."